### PR TITLE
Added correct loading logic for files and assets

### PR DESCRIPTION
**Pull request by ghost**
Right now, get files is triggered every time props is updated (hence it gets called a lot). this makes sure its only updated when assetId changes.

### CODE REVIEW

File: src/containers/AssetViewer.tsx [hunk 1]
```diff
@@ -69,55 +69,56 @@ export class AssetViewer extends React.Component<Props, State> {
...
+    if (assetId) {
+      return assets.all[assetId];
     }
     return undefined;
   }
 
   getNodeId = (fetchIfMissing: boolean) => {
     const {
-      app: { modelId, revisionId, nodeId },
+      app: { modelId, revisionId, nodeId, assetId },
```
Comment (dragly) [inline src/containers/AssetViewer.tsx]: Change to this.props.app

**Review by dragly — CHANGES_REQUESTED** [review]

File: src/containers/AssetViewer.tsx (hunk 1 above)
Comment (dlcognite) [reply src/containers/AssetViewer.tsx]: fixed

**Review by anderscognite — APPROVED** [review]

**Comment by cognite-cicd** [comment]
[pr-server]
View this change on [PR Server](https://pr-45.discovery.preview.cogniteapp.com)

**Comment by cognite-cicd** [comment]
[pr-server]
View this change on [PR Server](https://pr-45.discovery.preview.cogniteapp.com)

**Comment by cognite-cicd** [comment]
[pr-server]
View this change on [PR Server](https://pr-45.discovery.preview.cogniteapp.com)

**Pull request merged**